openhtf/io/output/print_console.py

```diff
@@ -0,0 +1,81 @@
+"""Module for outputting test record to JSON-formatted files."""
+import os
+import sys
+from json import JSONEncoder
+from openhtf import util
+
+
+class PrintConsole(JSONEncoder):
+  """Return an output callback that writes JSON Test Records.
+
+  An example filename_pattern might be:
+    '/data/test_records/%(dut_id)s.%(start_time_millis)s'
+
+  To use this output mechanism:
+    test = openhtf.Test(PhaseOne, PhaseTwo)
+    test.AddOutputCallback(openhtf.OutputToJson(
+        '/data/test_records/%(dut_id)s.%(start_time_millis)s'))
+
+  Args:
+    filename_pattern: A format string specifying the filename to write to,
+      will be formatted with the Test Record as a dictionary.
+    inline_attachments: Whether attachments should be included inline in the
+      output.  Set to False if you expect to have large binary attachments.
+  """
+
+  def __init__(self, filename_pattern=None, inline_attachments=True, **kwargs):
+    super(PrintConsole, self).__init__(**kwargs)
+    self.FAIL =  '\033[91m'
```

Comment:
I was about to work on this so I have some of the code written. Here's what I've done for coloring, you can decide which you think is better. You've already written it and it has bold output, so yours may be better.

```py
from tornado import log as tornado_log

for handler in logging.getLogger('openhtf').handlers:
  if isinstance(handler, logging.StreamHandler):
    handler.setFormatter(tornado_log.LogFormatter())

# That puts the formatter on 'openhtf', but we may want it just on 'openhtf.output.console'

LOG = logging.getLogger('openhtf.output.console')
LOG.propagate = False
handler = logging.StreamHandler()
handler.setFormatter(tornado_log.LogFormatter())
LOG.addHandler(handler)
```

Then in `__call__` you can do this:

```py
if result == 'PASS':
  func = LOG.info  # green
elif result == 'FAIL':
  func = LOG.error  # red
else:
  func = LOG.warning  # yellow

func('This is what you print')
```